LibZone/LibZone.lua

```diff
@@ -183,6 +342,10 @@ function lib:GetAllZoneDataById(reBuildNew, doReloadUI)
     --d(">languageIsMissingInTotal: " ..tostring(languageIsMissingInTotal))
     --Loop over all zone Ids and get it's data + name
     local addedAtLeastOne = false
+
+	-- Create the table used to add poiInfo to zoneData
+	buildPoiDataTable()

```

Comment:
I have it nil out the table since it is only needed during build. 
I did not know it was called so often. 
That can be change. can build on load and not nil it.

Reply:
What I meant is: This function lib:GetAllZoneDataById here is called for each event_add_on_loaded which will be at login, reloadui, zone change with loading screen afaik.
If the data you have added can be stored fixed at the LibZone_Data.lua tables move it there. If it needs to be updated with ingame API during event_Add_on_loaded then leave it here. Depends on what time it takes to scan that and if there will be a noticable lag then.

lib:GetAllZoneDataById can only be called manually by addons but shouldn't to be fair.

Reply:
I thought it was only ran OnLibraryLoaded unless specifically called by an addon. 
Really, the only time the table needs to exist is when updating data in the sv. It's likely close to being the same size of the SV file prior to the change. I figured that if it's not needed then dump it. The table itself does not take long to build. I've had 2 or 3 of them running on load from different sources. 
It's something worth testing further.